src/Features/Core/Portable/CodeRefactorings/PredefinedCodeRefactoringProviderNames.cs

```diff
@@ -48,5 +48,30 @@ internal static class PredefinedCodeRefactoringProviderNames
         public const string UseExpressionBody = "Use Expression Body Code Action Provider";
         public const string UseImplicitType = "Use Implicit Type Code Action Provider";
         public const string Wrapping = "Wrapping Code Action Provider";
+        public const string MakeLocalFunctionStatic = nameof(MakeLocalFunctionStatic);
+        public const string GenerateComparisonOperators = nameof(GenerateComparisonOperators);
```

Comment:
These are not in the same style as previous Names. Would be happy to standardize on either format if requested.

Reply:
Presumably these aren't displayed anywhere?

Reply:
As in they just use `nameof` instead spacing? I think it's fine personally. These are not customer facing names, just identifiers for us to use and read.

Reply:
Yeah, I figured they weren't displayed anywhere too exciting since they're not localized, but I thought maybe they were possibly shown in a yellow bar, where localization would be annoying, but perhaps spaces were the convention.